shell/testing/BUILD.gn

```diff
@@ -22,7 +22,7 @@ executable("testing") {
     "//third_party/dart/runtime:libdart_jit",
     "//third_party/dart/runtime/bin:embedded_dart_io",
     "//third_party/dart/runtime/bin:libdart_builtin",
-    "//third_party/dart/runtime/platform:libdart_platform",
```

Comment:
I had removed this without any problem in my local testing. Does it need to be there in the Fuchsia build?